controllers/admin/AdminCmsContentController.php

```diff
@@ -138,32 +138,32 @@ public function renderPageHeaderToolbar()
 
         if ($this->display == 'edit_category') {
             if (Tools::getValue('addcms_category') !== false) {
-                $this->toolbar_title[] = $this->l('Add new');
+                $this->toolbar_title[] = $this->trans('Add new', array(), 'Admin.Global');
             } else {
-                $this->toolbar_title[] = sprintf($this->l('Edit: %s'), $cms_category->name[$this->context->employee->id_lang]);
+                $this->toolbar_title[] = $this->trans('Edit: %name%', array('%name' => $cms_category->name[$this->context->employee->id_lang]), 'Admin.Parameters.Feature');
             }
         } elseif ($this->display == 'edit_page') {
             $this->toolbar_title[] = $cms_category->name[$this->context->employee->id_lang];
 
             if (Tools::getValue('addcms') !== false) {
-                $this->toolbar_title[] = $this->l('Add new');
+                $this->toolbar_title[] = $this->trans('Add new', array(), 'Admin.Global');
             } elseif ($id_cms_page) {
                 $cms_page = new CMS($id_cms_page);
-                $this->toolbar_title[] = sprintf($this->l('Edit: %s'), $cms_page->meta_title[$this->context->employee->id_lang]);
+                $this->toolbar_title[] = $this->trans('Edit: %meta_title%', array('%meta_title%' => $cms_page->meta_title[$this->context->employee->id_lang]), 'Admin.Parameters.Feature');
```

Comment:
I'd use Admin.Design wherever you used Admin.Parameters: this is CMS/Pages related, it's part of the Design section, not in the Parameters.

Also, former Admin.Parameters domain has been divided in Admin.ShopParam. and Admin.AdvParam to be more specific about the sections it relates too. So Admin.Parameters should no longer be used. I will publish an article on Build to let people know about these updates ^^